Pipfile

```diff
@@ -6,4 +6,3 @@ python_version = "2.7"
 "textblob" = "==0.15.1"
 "jsonschema" = "==2.6.0"
 "pandas" = "==0.21.0"
-w
```

Comment:
Does this have some significance? @PikkaPikkachu

Reply:
Yes